numpy/linalg/_linalg.py

```diff
@@ -1703,8 +1707,13 @@ def svd(a, full_matrices=True, compute_uv=True, hermitian=False):
 
     Examples
     --------
-    >>> a = np.random.randn(9, 6) + 1j*np.random.randn(9, 6)
-    >>> b = np.random.randn(2, 7, 8, 3) + 1j*np.random.randn(2, 7, 8, 3)
+    >>> rng = np.random.default_rng()
+    >>> a_re = rng.standard_normal(size=(9, 6))
+    >>> a_im = rng.standard_normal(size=(9, 6))
+    >>> a = a_re + 1j*a_im
+    >>> b_re = rng.standard_normal(size=(2, 7, 8, 3))
+    >>> b_im = rng.standard_normal(size=(2, 7, 8, 3))
+    >>> b = b_re + 1j*b_im
```

Comment:
This is a bit awkward, but I am not sure how to more cleanly create `a` and `b`

Reply:
I agree completely.  I tried multiple ways to rewrite this part, and all were worse that the above. I'll give `norm` a try and see if that improves it.

Reply:
For anyone reading this in the future, note that the new version is `normal` rather than `norm`.  Too many functions too keep track of.

Reply:
The shorter function name kept the line length to 77, so it's back to the shorter version.